plugins/de/opendiabetes/vault/plugin/importer/crawler/Authentication.java

```diff
@@ -0,0 +1,77 @@
+package de.opendiabetes.vault.plugin.importer.crawler;
+
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+
+/**
+ * Handles the login detail page.
+ */
+public class Authentication {
+
+    /**
+     * Holds the login cookies set by the webpage.
+     */
+    private Map<String, String> loginCookies = new HashMap<String, String>();
+
+    /*/**
+     * Useragent that will be sent to the webpage.
+     */
+    /*public static final String USERAGENT =
+            "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36";*/
+
+    /**
+     * Returns the login cookies set by the webpage.
+     *
+     * @return the login cookies hashmap.
+     */
+    Map<String, String> getcookies() {
```

Comment:
Sichtbarkeit?

Reply:
paketsichtbar (ohne Modifizierer) [https://stackoverflow.com/questions/215497/in-java-difference-between-package-private-public-protected-and-private] (https://stackoverflow.com/questions/215497/in-java-difference-between-package-private-public-protected-and-private) ?

Reply:
Okay, passt dann meiner Meinung nach